src/components/pages/Registration/InstructorWelcome.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '../../../styles/registration.less';
+
+export default function InstructorWelcome() {
+  return (
+    <div className="reg-content-container">
+      <div className="content">
+        <p className="color-one">Welcome to CoderHeroes!</p>
+        <p className="color-two">
+          You are steps away from sharing your superpowers with future
+          CoderHeroes
+        </p>
+        <a href="#">CREATE ACCOUNT</a>
```

Comment:
Not sure if I am the only one seeing it but I get a linting warning from Github, might be worth visiting before merging this PR.

Reply:
I see there are multiple now, and prob a recurring and okay theme throughout these merges/ PRs